lib/jira/resource/agile.rb

```diff
@@ -48,6 +48,9 @@ def path_base(client)
         self.class.path_base(client)
       end
 
+      def self.hash_to_query_string(query_params)
```

Comment:
I would suggest to avoid duplicate the code and use the one from the super class
https://github.com/sumoheavy/jira-ruby/blob/master/lib/jira/base.rb#L506